.vscode/launch.json

```diff
@@ -4,6 +4,19 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
Please put this file in .gitignore.

Reply:
Hi Athena I added that configuration to make life easy to test current opened spec file.

Reply:
The .vscode/launch.json file typically contains personal preferences for debugging configurations. Different developers may have distinct setups, configurations, or paths that work best for their local environments, which may not translate well across other machines.